snuba/utils/streams/kafka.py

```diff
@@ -658,36 +658,30 @@ def __init__(
         self.__commit_log_topic = commit_log_topic
         self.__group_id = configuration["group.id"]
 
-    def poll(self, timeout: Optional[float] = None) -> Optional[Message[TPayload]]:
-        self.__producer.poll(0.0)
-        return super().poll(timeout)
```

Comment:
was this not used ? Otherwise is it safe not to poll on the __producer anymore ?